### DISCUSSION TITLE
Unable to install using the installation command given "pip install "git+https://github.com/facebookresearch/pytorch3d.git""

### DISCUSSION THREAD

**Discussion by sandun21** [facebookresearch/pytorch3d#1725] [Q&A]
When I tried to install pytorch3d on my computer which is running linux with torch version=2.2  It gives an error saying
"error: command '/usr/bin/x86_64-linux-gnu-gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pytorch3d
  Running setup.py clean for pytorch3d
Failed to build pytorch3d"

**Comment by bottler** [comment]
What were the commands you used to build? Can you make them more verbose?
